src/freenas/etc/systemd/system/collectd.service.d/override.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
```

Comment:
Does this file accept comments? Due to nature of this it probably grants a couple lines explaining why this was needed, and link to a couple tickets